validator/testcases/regex.py

```diff
@@ -567,6 +568,20 @@ def tests(self):
             compat_type='error')
 
 
+@register_generator
+class Gecko54RegexTests(CompatRegexTestHelper):
+    VERSION = FX54_DEFINITION
+
+    def tests(self):
+        yield self.get_test_bug(
+            1333482,
+            r'-moz-appearance\b',
+            '-moz-appearance can only be used in chrome stylesheets.',
+            'All other users will be ignored.',
```

Comment:
Fair point, I honestly only copied from the issue. I feel like it should have been `All other uses will be ignored` which makes sense, let's stick to that?